src/main/kotlin/pl/put/smartgarden/infra/api/UserController.kt

```diff
@@ -0,0 +1,125 @@
+package pl.put.smartgarden.infra.api
+
+import io.swagger.annotations.*
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.*
+import pl.put.smartgarden.domain.user.UserService
+import pl.put.smartgarden.domain.user.dto.*
+import javax.validation.Valid
+
+@Api(description = "Users api")
+@RestController
+@RequestMapping("/users")
+class UserController(val userService: UserService) {
```

Comment:
we wszystkich metodach z 1 operacją możesz zamienić return na "=", chyba że będziemy tutaj coś logować sobie

Reply:
Akurat to mi się nie podoba w składni Kotlina, kod traci na przejrzystości według mnie :)